src/dashboard/src/templates/archival_storage/list.html

```diff
@@ -1,5 +1,6 @@
 {% extends "layout_fluid.html" %}
 {% load breadcrumb %}
+{% load i18n %}
```

Comment:
Can the i18n changes be in their own commit please? They're conceptually unrelated to the metadata DIP upload.